essay.tex

```diff
@@ -0,0 +1,43 @@
+
+Over the course of a sprint the need for control decreases.  The role of the scrum master mostly revolves around removing impediments to the process.  With user stories clearly assigned and in progress the Agile principle of trust must be adhered to.  Unless there are changing requirements to the projects parameters a soft touch can now be offered.  This is the point where capable staff thrive and as such, more demanding tasks should be delegated.  As the end of the sprint approaches there must be vigilant oversight to avoid overrunning or under achieving.  As deadlines near, unforeseen circumstances can have drastic implications and correctly identifying and managing these requires the autocratic strength of more dominant techniques. 
+
+\includegraphics{agile_graph.png}
+
+\section*{Conclusion}
+
+Given that the leadership style of scrum master can be transitioned from one style to another it is possible to optimise an Agile game development team's management through identifying the correct choice for scrum master dependant on the sprint goals and timeframe.  Despite being self-organising by nature, agile teams rely upon guidance beyond user stories, and differing coaching styles must be used to adjust for late-changing requirements and changes in focus and pressures over the course of a sprint.  By utilising the contrasting strengths offered by different management styles the process can be made more efficient, team morale can be improved, and overrunning time frames can be better avoided. 
```

Comment:
This conclusion doesn't link to the question very well. An explicitly stated question in the introduction and explicit link back to the opening will aid readers in understanding just what your answer is.